### ISSUE TITLE
Please set separate `about:` for openjdk

### ISSUE THREAD

**Issue by h-vetinari** [conda-forge/graalpy-feedstock#4]
Since this feedstock has an output that coincides with the main builds from https://github.com/conda-forge/openjdk-feedstock, this cause the description of that package to be wrong, see https://github.com/conda-forge/openjdk-feedstock/issues/116.

Please add a separate `about:` to the `openjdk` output which matches the summary from that feedstock (which IIUC correctly, should be enough to ensure that the artefacts get the correct description again).

**Comment by timfel** [comment]
I can copy the about section, but isn't the issue already with the openjdk-feedstock itself? Right now when I visit https://anaconda.org/conda-forge/openjdk I see

![image](https://github.com/conda-forge/graalpy-feedstock/assets/46235/531475aa-1f23-454a-910c-781772a28143)

So I guess the non-Linux build was uploaded last. But when I actually get an OpenJDK on Linux, I will not get a Zulu build at all. 🤔

**Comment by h-vetinari** [comment]
Yeah, I guess whichever feedstock has the most recent upload "wins". For some reason, the most recent graalvm openjdk build was uploaded almost 5 months ago, probably because its version and build-string haven't changed since then (so the CI here will not re-upload an artefact that matches an already existing one in that respect, even though the content here might be different).

Seeing that there have been new `graalpy` builds and the openjdk here (transitively) pins that package exactly, you probably have a broken wrapper package, and need to ensure that whenever the version (or build hash[^1]) changes, that the wrapper gets at least a a new hash, if not new build number/version.

Sidenote: for the same reason as `openjdk` here, the `python` should also get an `about:` that matches what python-feedstock does.

[^1]: e.g. if the versions of one of the host dependencies changes; this is at least is relatively easy to "pull" into the wrapper package by mirroring the host-dependencies of whatever you're wrapping.

**Comment by timfel** [comment]
> Sidenote: for the same reason as openjdk here, the python should also get an about: that matches what python-feedstock does.

PyPy also provides a `python` output, and no separate `about:` section for it (https://github.com/conda-forge/pypy-meta-feedstock/blob/main/recipe/meta.yaml). So are we saying all packages that provide matching outputs need to copy their about sections around? That seems suboptimal :-/

**Comment by timfel** [comment]
There something else going on with that website. Right now the latest build is a graalvm openjdk from this feedstock here:
![image](https://github.com/conda-forge/graalpy-feedstock/assets/46235/5eb93381-8d82-4e6d-803b-3c621ff5beb4)

But the description is
![image](https://github.com/conda-forge/graalpy-feedstock/assets/46235/9fdfa0a9-c043-4dc1-8d5b-2803d2c0aab9)

Which is not even the description for the previous linux uploads of the openjdk package. So I'm not convinced about how this works.
